index.js

```diff
@@ -24,67 +26,81 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
+  let guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
+  if (guess < 1) {
+    // Display a message for guesses lower than 0
+    numberLessThanZero.style.display = "";
+    return;
+  }
+
+  if (guess > 99) {
+    // Display a message for guesses higher than 99
+    numberHigherThan99.style.display = "";
+    return; // Exit the function to prevent further execution
+  }
+
   hideAllMessages();
 
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
 
-    correctMessage.style.display = '';
+    correctMessage.style.display = "";
 
     submitButton.disabled = true;
     guessInput.disabled = true;
-  }
-
-  if (guess !== targetNumber) {
+  } else {
     if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
+      tooLowMessage.style.display = ""; //Fixed the display message
```

Comment:
The comments are fine for this assignment, but on a real project, once a bug is fixed, you wouldn't need to leave a comment indicating that there was a bug in the past. Comments are good for explaining the way the code works now.